test-integration/test_integration/test_build.py

```diff
@@ -118,3 +118,24 @@ def test_build_gpu_model_on_cpu(tmpdir_factory, docker_image):
         }
     }
     assert "org.cogmodel.openapi_schema" not in labels
+
+
+def test_build_with_cog_init_templates(tmpdir_factory, docker_image):
+    tmpdir = tmpdir_factory.mktemp("project")
```

Comment:
Nit: If you're only using one temp dir you can use the [`tmpdir`](https://docs.pytest.org/en/6.2.x/tmpdir.html#the-tmpdir-fixture) fixture.

Reply:
Good catch. I updated all the other tests in that file to use `tmpdir` too: https://github.com/replicate/cog/pull/493/commits/415054b932a02637776b211c1611a4b884995cdf